src/main/java/com/github/maven_nar/Msvc.java

```diff
@@ -18,7 +18,6 @@
 import org.apache.maven.plugin.MojoFailureException;
 import org.apache.maven.plugins.annotations.Parameter;
 import org.apache.tools.ant.types.Environment.Variable;
-import org.codehaus.plexus.component.configurator.converters.basic.DoubleConverter;
```

Comment:
Would be better to just squash this into the previous commit using `git rebase -i HEAD^^` and changing the second line from `pick` to `fixup`